### ISSUE TITLE
Create a dedicated github org for this repo

### ISSUE THREAD

**Issue by nejch** [logchange/SecScanner2JUnit#49]
**Describe the idea**

Hey @angrymeir @marwin1991, nice project here :bow:

This is mostly cosmetics and maybe for visibility, but was wondering if you were open to creating a little for github org for this repo with both of you as admins/collaborators. We have users on our platform who have started using the Docker image, and we usually try to vet all images for our shared runners.

Usually those in personal namespaces tend to have a bit more of a risk of going unmaintained if the main author moves on to something else, so this is just for continuity. But I see there's been activity here recently as well so obviously not the case ;)

Even just something like https://github.com/SecScanner2JUnit/SecScanner2JUnit or something with kebab-case, just an idea here. :grinning:

**Comment by marwin1991** [comment]
Its @angrymeir decision, as creator of this awesome project 💟 

Anything I can do is to offer my simply team space,
https://github.com/logchange where I host community projects.

@angrymeir if you will decide, let me know so I will make you Owner of the team.

**Comment by angrymeir** [comment]
I agree, that a dedicated space would make more sense, especially since I'm almost always the bottleneck in the current setup!
Both options are fine with me. Since you did most of the work in this project I'll leave it up to you to decide @marwin1991 :)

**Comment by marwin1991** [comment]
@angrymeir I have invited you with Owner role to https://github.com/logchange org. Feel free to transfer project.

There are configured variables to all project to allow deploy to DockerHub

https://github.com/organizations/logchange/settings/secrets/actions

Checkout:
- https://github.com/logchange/valhalla/blob/master/.github/workflows/docker.yml
- https://github.com/logchange/valhalla/blob/master/.github/workflows/dockerhub-description.yml

**Comment by angrymeir** [comment]
Thank you @marwin1991, I'll move everything once I have a couple quite hours to ensure everything continues to work :)

**Comment by marwin1991** [comment]
Hello @angrymeir I think we can release one more version with information in logs to switch to new docker image also :)

**Comment by marwin1991** [comment]
@angrymeir hello, do you need some help on that? 

Best regards and have a nice day!
peter

**Comment by marwin1991** [comment]
@nejch done, https://hub.docker.com/repository/docker/logchange/secscanner2junit/general
